docs/services/navigation/_index.md

```diff
@@ -34,7 +34,6 @@ Click **Create service**:
 {{% tab name="JSON Template" %}}
 
 ```json {class="line-numbers linkable-line-numbers"}
-// 
 {
```

Comment:
```suggestion
// "services": [  ...
{
```
For whoever reviews this; would something like the above or `// <INSERT SERVICES JSON>` or similar be good for making it more clear what users need to do here?

Reply:
I like the way it is in https://docs.viam.com/services/vision/detection/ ? Have a look and see if you like that or if that gives you other ideas